srcpkgs/avrdude/template

```diff
@@ -1,9 +1,13 @@
 # Template file for 'avrdude'
 pkgname=avrdude
-version=6.4
+version=7.0
 revision=1
 build_style=gnu-configure
-configure_args="$(vopt_enable gpio linuxgpio) $(vopt_enable spi linuxspi)"
+configure_args="
+ $(vopt_enable spi linuxspi)
+ $(vopt_enable gpio linuxgpio)
+ $(vopt_enable parport parport)
+"
```

Comment:
Could you explain the reason for these build options instead of just enabling them all by default?

Reply:
The build options are already explained below, in these variables -
```sh
desc_option_spi="Enable Linux SPI support"
desc_option_gpio="Enable Linux sysfs GPIO support"
desc_option_parport="Enable parallel port support"
```